src/containers/JobDetails/JobDetails.test.js

```diff
@@ -223,66 +141,77 @@ describe('JobDetails', () => {
         business_state: 'WA',
         business_zip_code: '99900',
         additional_info: 'Amazon sucks',
-        job_id: '12345',
-        status: 'NOI filed'
+        job_id: '12345'
       }
     }
     })
-
+  
+    const mockBack = jest.fn()
+  
+  
     render(
       <Provider store={ store }>
         <MemoryRouter>
-          <JobDetails data={store.jobInfo}  key={'12345'}/>
+          <JobDetails data={store.jobInfo}  key={'12345'} onClick={mockBack}/>
         </MemoryRouter>
       </Provider>
     )
+  
+    const backButton = screen.getByRole('button', {name: 'Back'});
+  
+    expect(backButton).toBeInTheDocument();
+  
+    fireEvent.click(backButton);
+    // await waitFor(() => expect(mockBack).toHaveBeenCalled())
 
-    const lienButton = screen.getByRole('button', {name: 'Submit Lien'})
-
-    expect(lienButton).toBeInTheDocument();
   });
-  it('should display release lien button when status is lien filed', () => {
+
+  it('should not display the Submit or Remove buttons if status is NOI Requested', () => {
```

Comment:
I think we could still have the remove button? Definitely something to ask client

Reply:
Yeah this is a question for the client. Because we don't have the admin side setup I thought it could be risky for the user to be able to remove a job if the admin has already started with some of the paperwork.